doc/proxy-metrics.md

```diff
@@ -23,8 +23,9 @@ incremented when the response stream ends.
 
 ### `response_latency_ms`
```

Comment:
Should we rename this to indicate it's time-to-first-byte?

Reply:
Since this would necessitate a larger change to various components that consume this metric, I think we should at least defer this decision. I don't particularly mind the _latency_ terminology, though I agree that the documentation should be made clearer in any case.

Reply:
I took another pass at the documentation in https://github.com/runconduit/conduit/pull/981/commits/b30391f8cb7773f045575e61e4f641f9f42c0a46

Reply:
I don't think we'll want to keep calling this `latency` in the long run. There's just too much fuzziness around what it means. That said, it is good enough for now (tm).

I *do* think that we should add `total_request_time_ns` back in at some point in the future though.